src/redist/targets/Crossgen.targets

```diff
@@ -49,8 +49,8 @@
       <!-- Add back the .NET Core assemblies in the Sdks folder -->
       <RemainingFiles Include="$(SdkOutputDirectory)Sdks\Microsoft.NET.Sdk\tools\netcoreapp*\**\*" />
       <RemainingFiles Include="$(SdkOutputDirectory)Sdks\NuGet.Build.Tasks.Pack\CoreCLR\**\*" />
-      <RemainingFiles Include="$(SdkOutputDirectory)Sdks\Microsoft.NET.Sdk.Razor\**\*" />
-      <RemainingFiles Include="$(SdkOutputDirectory)Sdks\Microsoft.NET.Sdk.Web\**\*" />
```

Comment:
👍 I was about to say there are ns2.0 targeting analyzer binaries in this SDK. I'm not sure what crossgen does to them.

Reply:
@brianrob I think there are going to be assemblies under these paths that target both .NET Core and full Framework.  For example, there are net46 and netcoreapp3.1 folders under Sdks\Microsoft.NET.Sdk.Razor\tasks in my 3.1 SDK install.

Reply:
Yup.  So this change will only target the netcoreapp* directories.